source/bin/tvm.rs

```diff
@@ -170,3 +180,4 @@ mod tests {
         );
     }
 }
+
```

Comment:
Actually, quoting https://github.com/rust-lang/rust/blob/master/src/doc/style/style/whitespace.md:
> No trailing whitespace at the end of lines or files.

I understand this is not required and even discouraged from Rust style guides.

Also, some other projects like Redox, llmv-rs, or Rust itself, are not using this convention.

So I might ask you to revert this commit please :-).